### PR TITLE
Cloud SIEM light theme

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds information about the new theme preference to these articles:
- https://help.sumologic.com/docs/cse/introduction-to-cloud-siem/
- https://help.sumologic.com/docs/cloud-soar/overview/
- https://help.sumologic.com/docs/platform-services/automation-service/about-automation-service/
- https://help.sumologic.com/docs/get-started/account-settings-preferences/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-71](https://sumologic.atlassian.net/browse/DOCS-71)

### CODE REVIEW

File: docs/cse/introduction-to-cloud-siem.md [hunk 1]
```diff
@@ -24,6 +24,26 @@ Watch the following micro lesson to learn how to get started using Cloud SIEM fo
 
 import Iframe from 'react-iframe'; 
 
+## Cloud SIEM user interface
+
+### Access Cloud SIEM
+
+To access Cloud SIEM, click **Cloud SIEM** in the Sumo Logic navigation menu. 
+
+Cloud SIEM must be enabled by Sumo Logic before it is accessible to users in your organization. For more information, see [Onboarding Checklist for Cloud SIEM Administrators](/docs/cse/get-started-with-cloud-siem/onboarding-checklist-cse/).
```
Comment (kimsauce) [inline docs/cse/introduction-to-cloud-siem.md:33]:
```suggestion
Cloud SIEM must be enabled by Sumo Logic before it is accessible to users in your organization. If you would like to use Cloud SIEM in your organization, contact your Sumo Logic account representative.
```

File: docs/cse/introduction-to-cloud-siem.md [hunk 2]
```diff
@@ -24,6 +24,26 @@ Watch the following micro lesson to learn how to get started using Cloud SIEM fo
...
+
+### Theme
+
+import Theme from '../reuse/theme.md';
+
+<Theme/>
+
+### Heads Up Display
+
+The first screen you see when you access Cloud SIEM is the Heads Up Display, a single pane of information about your environment. In the center you see a radar showing insights in the center, surrounded by the signals and records used to generate the insights. On the left is summary information, and on the right is recent activity. Use this screen as the starting place for your investigations, focusing on insights as the most valuable place to start.
```
Comment (kimsauce) [inline docs/cse/introduction-to-cloud-siem.md:43]:
```suggestion
The first screen you see when you access Cloud SIEM is the Heads Up Display, a single pane of information about your environment.  Use this screen as the starting place for your investigations, focusing on insights as the most valuable place to start. 

In the center, you'll see a radar showing insights, surrounded by the signals and records used to generate the insights. On the left is summary information, and on the right is recent activity.
```

File: docs/get-started/account-settings-preferences.md [hunk 3]
```diff
@@ -11,51 +11,57 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 You can review and update your account settings and login credentials at any time in your user **Preferences**. The **Preferences** page contains settings that apply to your account only and do not affect other users in your organization.
 
+To access preferences, click your user name in the navigation menu and select **Preferences**.
```
Comment (kimsauce) [inline docs/get-started/account-settings-preferences.md:14]:
```suggestion
To access preferences, click your user name in the Sumo Logic navigation menu and select **Preferences**.
```

File: docs/get-started/account-settings-preferences.md [hunk 4]
```diff
@@ -118,22 +126,33 @@ Keep this option selected if you'd like to run a saved search as soon as you sel
...
 
-### Query editing
+### Theme
+
+Select to show the user interface in a **Dark theme** or **Light theme**.
+
+This setting only applies to the user interface in the [Automation Service](/docs/platform-services/automation-service/), [Cloud SIEM](/docs/cse/), and [Cloud SOAR](/docs/cloud-soar/).
+
+:::note
+You can also set a dark them or light theme for dashboards. See  [Dark Theme](/docs/dashboards/about/#dark-theme) in *About Dashboard*.
```
Comment (kimsauce) [inline docs/get-started/account-settings-preferences.md:144]:
```suggestion
You can also set a dark them or light theme for dashboards. See [Dark Theme](/docs/dashboards/about/#dark-theme) in *About Dashboard*.
```

File: docs/reuse/theme.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+You can view the user interface in a dark theme or a light theme:
```
Comment (kimsauce) [inline docs/reuse/theme.md:1]: maybe we could title this file something like `dark-light-theme.md` to make it easier to find down the road

**Review by kimsauce — APPROVED** [review]
Some suggestions!

File: docs/reuse/theme.md (hunk 5 above)
Comment (jpipkin1) [reply docs/reuse/theme.md]: Excellent idea, @kimsauce! Done.

**Comment by jpipkin1** [comment]
Thanks for the great suggestions, @kimsauce!

**Pull request merged**